client/src/lib/result.js

```diff
@@ -40,3 +47,68 @@ export function paddedAttemptResults(result, numberOfAttempts) {
   const attemptResults = result.attempts.map((attempt) => attempt.result);
   return padSkipped(attemptResults, numberOfAttempts);
 }
+
+// Events sorted by best don't need forecast view.
+// Fewest moves is currently unsupported
+// Only final rounds are supported
+export function forecastViewEnabled(round) {
+  return round.format.sortBy != "best" &&
+    round.competitionEvent.event.id != "333fm" &&
+    round.name === "Final";
```

Comment:
The easiest way to distinguish is to check there is no advancement condition :D

```suggestion
    round.advancementCondition === null;
```